### DISCUSSION TITLE
What is the maxmimum amount of links allowed per account?

### DISCUSSION THREAD

**Discussion by roke-julian-lockhart** [dubinc/dub#392] [Q&A] [answered]
If one doesn't exist, that's great. I don't want to commit to this if I have to pay something after 5 links, for instance.

**Comment by gabefletch** [comment, marked as answer]
All plans have an unlimited number of  `dub.sh` links, its the number of clicks that is limited by your plan:<br>
- Up to 1K clicks/month on Free plan
- Up to 50K clicks/month on Pro
- Unlimited clicks on Enterprise

More information on what limits exist for each plan can be found at [dub.co/pricing](https://dub.co/pricing).
